### PR TITLE
feat: collect event streams used by running activations

### PR DESCRIPTION
**Pull request by hsong-rh**
https://issues.redhat.com/browse/AAP-36177

Collect those event streams that are currently used by activations

### CODE REVIEW

File: tests/integration/analytics/test_analytics_collectors.py [hunk 1]
```diff
@@ -575,31 +624,73 @@ def test_event_streams_table_by_activation_collector(
...
+@pytest.mark.django_db
+def test_event_streams_table_by_running_activations_collector(
+    activations_with_different_status: list[models.Activation],
+    default_event_streams: list[models.EventStream],
+):
+    activations_with_different_status[0].event_streams.add(
+        default_event_streams[0]
+    )
+    activations_with_different_status[1].event_streams.add(
+        default_event_streams[1]
```
Comment (bzwei) [inline tests/integration/analytics/test_analytics_collectors.py:641]: Can you add a case that an event stream is used by multiple activations? Can it be deduplicated?

File: src/aap_eda/analytics/analytics_collectors.py [hunk 2]
```diff
@@ -478,6 +478,57 @@ def event_streams_by_activation_table(
...
+    event_streams = models.EventStream.objects.none()
+    for activation in activations:
+        event_streams |= activation.event_streams.all()
+
+    if not bool(event_streams):
+        return
+
+    event_streams = event_streams.annotate(
+        event_stream_id=F("id"),
+        activation_id=F("activations__id"),
```
Comment (Alex-Izquierdo) [inline src/aap_eda/analytics/analytics_collectors.py:513]: This query will duplicate the event stream record per activation related if there are multiple activations using the same event stream. Is this the desired outcome or maybe we want only one record per event stream with a list of the activation ids consuming it? 

I don't have context about this data will be consumed, I'm not saying it is wrong as it is, I'm just making sure.

File: tests/integration/analytics/test_analytics_collectors.py (hunk 1 above)
Comment (hsong-rh) [reply tests/integration/analytics/test_analytics_collectors.py:641]: Yes, even streams used by different activations are treated as other records in the collection table.

File: src/aap_eda/analytics/analytics_collectors.py (hunk 2 above)
Comment (hsong-rh) [reply src/aap_eda/analytics/analytics_collectors.py:513]: Yeah, the analytics team can easily use the flat list with other data to combine the final result.

**Review by bzwei — APPROVED** [review]

**Review by dhaustein — APPROVED** [review]

**Pull request merged**